### ISSUE TITLE
Google Drive login fails in Brave browser (Chromium)

### ISSUE THREAD

**Issue by khenzar** [Authenticator-Extension/Authenticator#373]
When trying to setup Backup Sync using Google Drive, the Sign-In fails with the following message:
![image](https://user-images.githubusercontent.com/2130285/63083829-0098a680-bf4a-11e9-866b-c0069cc769ee.png)

Additional info:
Dropbox login works fine. Google login works fine on other websites and extensions.

**Platform:**
 - Browser: [Brave]
 - Browser Version: 0.67.125 Chromium: 76.0.3809.100 (Official Build) (64-bit)

**Comment by mymindstorm** [comment]
Try to login, do https://authenticator.cc/docs/en/debug-info, type `window.navigator.userAgent` into the console, hit enter, and finally copy paste all messages to the issue desc. 

Brave is not officially supported, the issue comes from using a chrome-specific API and Brave having Chrome in the useragent.

**Comment by Kukks** [comment]
Clicking on Sign in ( or [screenshot if easier to read](https://i.imgur.com/tSphen6.png)):
```
backup.ts:105 GET https://www.googleapis.com/drive/v3/files 401
(anonymous) @ backup.ts:105
(anonymous) @ backup.ts:73
(anonymous) @ otp.ts:5
__awaiter @ otp.ts:5
getToken @ backup.ts:71
(anonymous) @ backup.ts:272
fulfilled @ otp.ts:5
Promise.then (async)
step @ otp.ts:5
(anonymous) @ otp.ts:5
__awaiter @ otp.ts:5
upload @ backup.ts:264
(anonymous) @ backup.ts:30
(anonymous) @ otp.ts:5
__awaiter @ otp.ts:5
backupUpload @ backup.ts:14
(anonymous) @ vue.min.js:6
(anonymous) @ popup.ts:87
popup.html:1 Unchecked runtime.lastError: The user turned off browser signin
backup.ts:105 GET https://www.googleapis.com/drive/v3/files 401
(anonymous) @ backup.ts:105
(anonymous) @ backup.ts:73
(anonymous) @ otp.ts:5
__awaiter @ otp.ts:5
getToken @ backup.ts:71
(anonymous) @ backup.ts:174
(anonymous) @ otp.ts:5
__awaiter @ otp.ts:5
getFolder @ backup.ts:173
(anonymous) @ backup.ts:278
fulfilled @ otp.ts:5
Promise.then (async)
step @ otp.ts:5
fulfilled @ otp.ts:5
Promise.then (async)
step @ otp.ts:5
(anonymous) @ otp.ts:5
__awaiter @ otp.ts:5
upload @ backup.ts:264
(anonymous) @ backup.ts:30
(anonymous) @ otp.ts:5
__awaiter @ otp.ts:5
backupUpload @ backup.ts:14
(anonymous) @ vue.min.js:6
(anonymous) @ popup.ts:87
popup.html:1 Unchecked runtime.lastError: The user turned off browser signin
backup.ts:255 POST https://www.googleapis.com/drive/v3/files/ 401
(anonymous) @ backup.ts:255
(anonymous) @ backup.ts:224
fulfilled @ otp.ts:5
Promise.then (async)
step @ otp.ts:5
(anonymous) @ otp.ts:5
__awaiter @ otp.ts:5
getFolder @ backup.ts:173
(anonymous) @ backup.ts:278
fulfilled @ otp.ts:5
Promise.then (async)
step @ otp.ts:5
fulfilled @ otp.ts:5
Promise.then (async)
step @ otp.ts:5
(anonymous) @ otp.ts:5
__awaiter @ otp.ts:5
upload @ backup.ts:264
(anonymous) @ backup.ts:30
(anonymous) @ otp.ts:5
__awaiter @ otp.ts:5
backupUpload @ backup.ts:14
(anonymous) @ vue.min.js:6
(anonymous) @ popup.ts:87
backup.ts:245 Invalid Credentials
xhr.onreadystatechange @ backup.ts:245
XMLHttpRequest.send (async)
(anonymous) @ backup.ts:255
(anonymous) @ backup.ts:224
fulfilled @ otp.ts:5
Promise.then (async)
step @ otp.ts:5
(anonymous) @ otp.ts:5
__awaiter @ otp.ts:5
getFolder @ backup.ts:173
(anonymous) @ backup.ts:278
fulfilled @ otp.ts:5
Promise.then (async)
step @ otp.ts:5
fulfilled @ otp.ts:5
Promise.then (async)
step @ otp.ts:5
(anonymous) @ otp.ts:5
__awaiter @ otp.ts:5
upload @ backup.ts:264
(anonymous) @ backup.ts:30
(anonymous) @ otp.ts:5
__awaiter @ otp.ts:5
backupUpload @ backup.ts:14
(anonymous) @ vue.min.js:6
(anonymous) @ popup.ts:87
backup.ts:327 POST https://www.googleapis.com/upload/drive/v3/files?uploadType=multipart 401
(anonymous) @ backup.ts:327
(anonymous) @ backup.ts:279
fulfilled @ otp.ts:5
Promise.then (async)
step @ otp.ts:5
fulfilled @ otp.ts:5
Promise.then (async)
step @ otp.ts:5
fulfilled @ otp.ts:5
Promise.then (async)
step @ otp.ts:5
(anonymous) @ otp.ts:5
__awaiter @ otp.ts:5
upload @ backup.ts:264
(anonymous) @ backup.ts:30
(anonymous) @ otp.ts:5
__awaiter @ otp.ts:5
backupUpload @ backup.ts:14
(anonymous) @ vue.min.js:6
(anonymous) @ popup.ts:87
backup.ts:306 Invalid Credentials
xhr.onreadystatechange @ backup.ts:306
XMLHttpRequest.send (async)
(anonymous) @ backup.ts:327
(anonymous) @ backup.ts:279
fulfilled @ otp.ts:5
Promise.then (async)
step @ otp.ts:5
fulfilled @ otp.ts:5
Promise.then (async)
step @ otp.ts:5
fulfilled @ otp.ts:5
Promise.then (async)
step @ otp.ts:5
(anonymous) @ otp.ts:5
__awaiter @ otp.ts:5
upload @ backup.ts:264
(anonymous) @ backup.ts:30
(anonymous) @ otp.ts:5
__awaiter @ otp.ts:5
backupUpload @ backup.ts:14
(anonymous) @ vue.min.js:6
(anonymous) @ popup.ts:87
```

```
window.navigator.userAgent
"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/76.0.3809.132 Safari/537.36"
```

**Comment by mymindstorm** [comment]
This can't be fixed if Brave doesn't put anything in it's user agent to differentiate it from Chrome.

**Comment by rodrigoswz** [comment]
I can confirm that it's fixed on Brave 1.15.72 (Chromium 86.0.4240.75)

You just need enable "Allow Google Login for extensions":
https://github.com/brave/brave-browser/wiki/Allow-Google-login---Third-Parties-and-Extensions
